Tasks/AzureCloudPowerShellDeploymentV1/task.json

```diff
@@ -24,6 +24,11 @@
     ],
     "minimumAgentVersion": "1.103.0",
     "groups": [
```

Comment:
Change minor version of the task to 155

Reply:
Once you increment the version in the `task.json` file. 

Then build the task using command : `node make.js build --task AzureCloudPowerShellDeploymentV1`  . It will update the minor version in `task.loc.json` file too